src/tasks.ts

```diff
@@ -20,7 +20,7 @@
 // Artifex Software, Inc., 39 Mesa Street, Suite 108A, San Francisco,
 // CA 94129, USA, for further information.
 
-import * as mupdf from "mupdf"
+import * as mupdf from "mupdf/core"
```

Comment:
```suggestion
import * as core from "mupdf/core"
```
I think we should rename it like this.